map/routing_manager.cpp

```diff
@@ -702,6 +720,12 @@ bool RoutingManager::InsertRoute(Route const & route)
           FillTurnsDistancesForRendering(segments, subroute->m_baseDistance, subroute->m_turns);
           break;
         }
+      case RouterType::Ruler:
+        {
+          subroute->m_routeType = df::RouteType::Ruler;
+          subroute->AddStyle(df::SubrouteStyle(df::kRouteRuler, df::RoutePattern(16.0, 2.0)));
```

Comment:
nit: Make pattern a recognizable constant to find and edit it easier?

Reply:
nit: We should provide constans for Pedestrian and Bycicle dashed lines too. Maybe move those parameters to MapCSS as we did with `RouteBicycle-color`, `RouteRuler-color`?

Reply:
That is a good idea!